src/components/Contacts/ContactFilters/ContactFilters.tsx

```diff
@@ -81,9 +112,11 @@ export const ContactFilters: React.FC<Props & BoxProps> = ({
   };
 
   const getSelectedFilters = (group: FilterGroup) =>
-    group.filters.filter((value) => selectedFilters[value.name]);
+    group.filters.filter(
+      (value) => selectedFilters[value.filterKey as ContactFilterKey],
```

Comment:
Does this need to be camel cased?

Reply:
Good catch, it probably does.

Reply:
I think this is still outstanding. Everything else looks good :)

Reply:
I think I made this change... could you check again?

Reply:
I missed that somehow... maybe I clicked the link to the outdated file... Looks good.